internal/server/middleware.go

```diff
@@ -157,37 +185,12 @@ func orgRequired() gin.HandlerFunc {
 
 		if org == nil {
 			sendAPIError(c, internal.ErrBadRequest)
-		}
-	}
-}
-
-func setOrganizationInCtx(srv *Server) gin.HandlerFunc {
-	return func(c *gin.Context) {
-		// TODO: use a transaction
-		org, err := getOrgFromRequest(c.Request, srv.DB())
```

Comment:
I have to remove this middleware to resolve this TODO because the middleware was called before we started a DB transaction.

This was originally a bug in the changes I made to merge in the `org_collab` branch.